source/routing/raptor_api.cpp

```diff
@@ -490,16 +476,15 @@ static void add_pathes(EnhancedResponse& enhanced_response,
             if (use_crow_fly(origin, departure_stop_point, sn_departure_path, d)){
                 type::EntryPoint destination_tmp(type::Type_e::StopPoint, departure_stop_point->uri);
                 destination_tmp.coordinates = departure_stop_point->coord;
-                bt::time_period action_period(path.items.front().departures.front(),
+                pb_creator.action_period = bt::time_period (path.items.front().departures.front(),
```

Comment:
In some case action_period value is re-calculated where as in others it's value does not change.